forecast-framework.qmd

```diff
@@ -66,10 +66,8 @@ extract_recipe(out_gb$epi_workflow)
 ```
 
 ```{r}
-#| message: true
 #| echo: false
-#| collapse: true
-extract_recipe(out_gb$epi_workflow)
+with_messages_cat_to_stdout(print(extract_recipe(out_gb$epi_workflow)))
```

Comment:
Generic comment:

with all these patterns, we lose the display of `extract_recipe()` (or similar) usage. Maybe we want to think about showing the feature in 1 chunk and printing with a second chunk?

Reply:
If I understand correctly, you want both code saying
```r
extract_recipe(........)
```
and the better-formatted output from `with_messages_.........` that doesn't turn it into a bunch of chunks?

I think current state of PR should do that.  Every replacement was of something that was already `echo: false`, except for `five_days_ahead$epi_workflow`, which was split into a `results: false` chunk + `echo: false` chunk.